packages/emoji-mart/tsconfig.json

```
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "target": "es2016",
```

All modern evergreen browsers & runtimes support at least ES2020 now. No need to transpile down beyond it.

```suggestion
    "target": "es2020",
```